### ISSUE TITLE
Help to recover BIP39 Phrase

### ISSUE THREAD

**Issue by rbwiz** [gurnec/btcrecover#491]
I know the correct order of the first 4 words, and the other 20 words are shuffled, can i modify the code the find my seed ?

**Comment by GarettBill** [comment]
Same thing happened to me but thanks to Blackhathacker111 who helped me,message him on telegram his telegram username is https://t.me/Blackhathacker111

**Comment by An1an1** [comment]
> Я знаю правильный порядок первых 4 слов, а остальные 20 слов перемешаны, могу ли я изменить код, чтобы найти мое семя?

можно попробовать, какие слова ?

**Comment by An1an1** [comment]
> I know the correct order of the first 4 words, and the other 20 words are shuffled, can i modify the code the find my seed ?

you can try, what are the words?

**Comment by 3rdIteration** [comment]
No, there are 20! (20 factorial) possible seeds, the math of it makes it impractical.

**Comment by rbwiz** [comment]
I see. Thank you for the reply.

**Comment by DrrDino** [comment]
> I know the correct order of the first 4 words, and the other 20 words are shuffled, can i modify the code the find my seed ?

I will advise you to contact the best hacker around he helped me recover my wallet,fast and reliable his telegram is https://t.me/Blackhathacker111
